skimage/feature/tests/test_texture.py

```diff
@@ -1,7 +1,13 @@
 import numpy as np
-from skimage.feature import greycomatrix, greycoprops, local_binary_pattern
-from skimage._shared.testing import test_parallel
+from skimage.feature import (
+                            greycomatrix,
```

Comment:
Usually we go ahead and populate the first object, indent the rest per PEP8, then place the end parenthesis after the final object.